voodoo-gen.opam.template

```diff
@@ -0,0 +1,3 @@
+pin-depends: [
+  ["pandoc.dev" "git+https://github.com/tatchi/opam-pandoc-bin#main"]
```

Comment:
We need the `pandoc` binary to make the conversion. To not rely on a globally installed binary, I created an opam package that contains the prebuilt binaries. Heavily inspired from https://github.com/tmattio/opam-esbuild by @tmattio